daemon/entrypoint.sh

```diff
@@ -276,7 +276,12 @@ function osd_directory {
   fi
 
   for OSD_ID in $(ls /var/lib/ceph/osd |  awk 'BEGIN { FS = "-" } ; { print $2 }'); do
-    if [ -n "${JOURNAL_DIR}" ]; then
+    OSD_JOURNAL_TMP="OSD_JOURNAL$OSD_ID"
+    OSD_JOURNAL=${!OSD_JOURNAL_TMP}
+    if [ -n "${OSD_JOURNAL}" ]; then
+      ceph-disk -v prepare ${OSD_JOURNAL}
```

Comment:
Agree with @Ulexus but to go back for a second to your `ceph-disk` usage. Perhaps it's me but if you run `ceph-disk prepare /dev/sdb1` you will just get a new OSD on this partition. In the end, this won't prepare a journal but create a new OSD. Am I missing something here?